astro-airflow-iris/{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -2,6 +2,7 @@ ipython>=8.10
 jupyterlab>=3.0
 notebook
 kedro~={{ cookiecutter.kedro_version }}
+kedro[jupyter]
```

Comment:
Would it make sense to update the line above to `kedro[jupyter]~={{ cookiecutter.kedro_version }}` itself?

Reply:
I don't feel strongly either tbh and that makes sense ^